docs/portal/accessing-portal/forgot-password.md

```diff
@@ -7,7 +7,7 @@ If you forgot your password:
 
 ![null](<../../resources/images/forgotpsword.png>)
 
-1. Click the **Submit** button.
+3. Click the **Submit** button.
```

Comment:
Hey Marc, do you think it's really necessary?

Reply:
I think so, thanks.